project/plugins.sbt

```diff
@@ -29,3 +29,5 @@ addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.1.6")
 libraryDependencies += "com.timgroup" % "java-statsd-client" % "2.0.0"
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.0.0-RC1")
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.1.8")
```

Comment:
No strong opinion, it's one of these that I see people usually have in their global settings. 
Can be removed I guess, each plugin adds to load time of the build.

:+1: the use of it anyway, it's a great plugin..